addons/point_of_sale/tests/test_frontend.py

```diff
@@ -604,10 +604,15 @@ def test_05_ticket_screen(self):
         self.main_pos_config.with_user(self.pos_user).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'TicketScreenTour', login="pos_user")
 
-    def test_product_information_screen_admin(self):
+    def test_misc_with_pos_admin(self):
```

Comment:
I'm not a fan of changing method signature in stable but I guess that it is ok since it is a test and the stable is out for only one day

Reply:
I restored the name.